charts/grafana/templates/_pod.tpl

```diff
@@ -857,6 +857,9 @@ containers:
     volumeMounts:
       - name: sc-plugins-volume
         mountPath: "/etc/grafana/provisioning/plugins"
+      {{- with .Values.sidecar.plugins.extraMounts }}
```

Comment:
please add this to all sidecars

Reply:
Sure, do you want me to remove the trim function from existing extraMount or add it to the new extraMount for the sake of consistency?

Reply:
Yes, add it for the sake of consistency.